.travis.yml

```diff
@@ -18,6 +18,8 @@ matrix:
     - env: LANGUAGE=php           VERSION=5.3  PHP_VERSION=5.3.10 PHP_BUILD_INSTALL_EXTENSION=apcu=4.0.11 REDIS_VERSION=3.1.6 SECURITY_CHECKER_VERSION=2.0.6
     - env: LANGUAGE=php           VERSION=5.6  PHP_VERSION=5.6.35-cli-alpine APCU_VERSION=4.0.11 COMPOSER_VERSION=1.6.3 REDIS_VERSION=4.0.0 SECURITY_CHECKER_VERSION=4.1.6 XDEBUG_VERSION=XDEBUG_2_5_5
     - env: LANGUAGE=php           VERSION=7.1  PHP_VERSION=7.1.16-cli-alpine APCU_VERSION=5.1.11 COMPOSER_VERSION=1.6.3 REDIS_VERSION=4.0.0 SECURITY_CHECKER_VERSION=4.1.8 XDEBUG_VERSION=2.6.0
+    - env: LANGUAGE=python        VERSION=2.7.15  PYTHON_VERSION=2.7.15
```

Comment:
`VERSION` should be `2.7` (same for `3.6`)

Reply:
Done